web-client/src/app.vue

```diff
@@ -34,80 +34,100 @@ form#patient_form(:class="{ [currentStep]: true }")
           .legend 100%
     #patient-medications
       #meds-with-rules Recommended measures:&nbsp;
-        a(v-for="({ ATC_code: href, medication_name: title }, index) in medicationAdvice",
```

Comment:
medication_name is always filled, but generic_name is not, thus I suspect this is wrong because I bet you'll need to access both

Reply:
added fallback deployed to ace-pr as well

Reply:
switched fallback to prefer mn on gn for display/title